### PR TITLE
[fix] fix edge some word

### PR DESCRIPTION
**Pull request by donghui12**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
fix edged word and variable，format error log
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edged/edged.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ var DefaultRunLiteKubelet RunLiteKubelet = kubeletserver.Run
 // edged is the main edged implementation.
 type edged struct {
 	enable      bool
-	KuberServer *kubeletoptions.KubeletServer
+	KubeServer  *kubeletoptions.KubeletServer
```
[the remark this answers is not in the document]
Comment (Shelley-BaoYue) [reply edge/pkg/edged/edged.go:85]: > kubeServer or kubeletServer cc @Shelley-BaoYue

kubeServer is ok.

**Comment by Shelley-BaoYue** [comment]
/lgtm
and you need run ci

**Comment by donghui12** [comment]
how to run ci

**Comment by gy95** [comment]
wait for maintainers to approve running CI workflows :)
/ok-to-test

**Comment by donghui12** [comment]
\cc @Qi Zhang @Dave Chen @Jie Zhang Please help to run ci

File: edge/pkg/edged/edged.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (fisherxu) [reply edge/pkg/edged/edged.go:85]: > kubeServer or kubeletServer

kubeletServer should be more better? @Shelley-BaoYue @gy95

**Comment by gy95** [comment]
LGTM
please squash commits to one, and fix DCO :)

**Review by fisherxu — COMMENTED** [review]
Looks like the lint ci reports err `edge/pkg/edged/edged.go:84: File is not `gofmt`-ed with `-s` (gofmt)`  @donghui12

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**